### ISSUE TITLE
why not use marker when the duration exceed 30s?

### ISSUE THREAD

**Issue by ILG2021** [EtienneAb3d/WhisperHallu#12]

**Comment by EtienneAb3d** [comment]
@ILG2021 
Two reasons:

1. to get a good marker recognition, the "prompt" texts are very useful. But, Whisper is processing audio files by segments of 30s. The "prompt" is used only on the first 30s. After 30s, without the "prompt" information, it is really less efficient to produce good marker outputs.
2. in my tests, when using noise and silence removal, it is sufficient to produce quite good results on files larger than 30s, without hallucination (and not sufficient on smaller files).

**Comment by ILG2021** [comment]
Yes, WhisperHallu is a very good solution for the silence sound which cause youtube ads. But it seem can not solve the repeat sentence problem. I have a Chinese fine tune faster whisper and when I speak two similar sentence, the model will produce repeat sentence. I don't know why. The result is：
